plugins/resizimg/resize.with.canvas.js

```diff
@@ -1,212 +1,218 @@
-function ResizeWithCanvas() {
+var ResizeWithCanvas = (function ($) {
```

Comment:
remove the `var ... =` part, it's not useful here ^^

Reply:
doh! 🙈

Sorry, but I missed a lot of javascript best practice.